client/checkout/classic/upe.js

```diff
@@ -192,6 +205,32 @@ jQuery( function ( $ ) {
 		}
 	};
 
+	/**
+	 * Converts form fields object into Stripe `billing_details` object.
+	 *
+	 * @param {Object} fields Object mapping checkout billing fields to values.
+	 * @return {Object} Stripe formatted `billing_details` object.
+	 */
+	const getBillingDetails = ( fields ) => {
+		return {
+			name: (
+				fields.billing_first_name +
+				' ' +
+				fields.billing_last_name
```

Comment:
(minor) we could use string a template here
```suggestion
				`${fields.billing_first_name} ${fields.billing_last_name}`
```

Reply:
Nice, I like it. Done,

Reply:
Thanks for considering it!
TBH, I haven't ensured that linter would be happy with it 👀